src/SignalRServiceExtension/TriggerBindings/Utils/SignalRTriggerUtils.cs

```diff
@@ -65,9 +66,9 @@ public static string GetAccessKey(string connectionString)
                 return default;
             }
 
-            // The claim string looks like "a= v, b= v"
-            return claims.Split(HeaderSeparator)
-                .Select(p => p.Split(ClaimsSeparator, StringSplitOptions.RemoveEmptyEntries))
+            // The claim string looks like "a: v, b: v"
+            return claims.Split(HeaderSeparator, StringSplitOptions.RemoveEmptyEntries)
+                .Select(p => p.Split(ClaimsSeparator, StringSplitOptions.RemoveEmptyEntries)).Where(l => l.Length == 2)
```

Comment:
>ClaimsSeparator [](start = 37, length = 15)

what if value contains `:`? will it destroy the structure?

Reply:
I use `: ` as a separator. But if there a `: ` in claim, it will be destroyed. Do we have other options?